generators/react/react.js

```diff
@@ -90,7 +90,7 @@ async function installACPaaSUI(config) {
   updateLog('Installing ACPaaS UI...');
 
   try {
-    await execPromise('npm', ['install', '--save-dev', 'node-sass'], { cwd: path.resolve('frontend') });
+    await execPromise('npm', ['install', '--save-dev', 'node-sass@4.8.3'], { cwd: path.resolve('frontend') });
```

Comment:
Zouden we niet beter volgend commando nemen (voor 't geval er nog andere patches gereleased worden op 4.x.x)?
```suggestion
    await execPromise('npm', ['install', '--save-dev', 'node-sass@"<5.0.0"'], { cwd: path.resolve('frontend') });
```

Reply:
er stonden een paar quotes te veel in uw suggestie maar ik heb in essentie het zelfde gecommit

Reply:
Vreemd, want volgens hetgeen ik opzocht, moesten de quotes erbij.  Maar ik ga ervan uit dat je het getest hebt ;-)